src/modules/getDataFood.js

```diff
@@ -0,0 +1,74 @@
+import getItemData from './createPopup.js';
+import hidenItems from './hidenItems.js';
+import { addLikeToItem, getLikesToItems } from './likeDataApi.js';
+
+// function display an array of objects
+const displayData = (arr) => {
+  const board = document.querySelector('.container-food-cards');
+  board.innerHTML = '';
+  arr.forEach((food) => {
+    const item = document.createElement('article');
+    item.id = food.idMeal;
+
+    item.classList.add('card-food');
+    item.innerHTML = `
+        <div class="card-title">
+          <h5>${food.strMeal}</h5>
+        </div>        
+        <button type="button" class="btn-recipe">Recipe</button>
+        <div>        
+        </div>
+        <div class="btn-liked">
+          Like
+        </div>
+        <div class="card-img">
+         <img src="${food.strMealThumb}" class="img-food">        
+        </div>     
+    `;
+
+    const btnRecipeElement = item.querySelector('.btn-recipe');
+    const btnLikedElement = item.querySelector('.btn-liked');
+    const printLike = (data) => {
+      const likesReturned = data.find(
+        // eslint-disable-next-line comma-dangle
+        (element) => element.item_id === food.idMeal
+      );
+      // eslint-disable-next-line operator-linebreak
+      btnLikedElement.innerHTML =
+        likesReturned !== undefined
+          ? `<ion-icon name="heart-outline"></ion-icon>(${likesReturned.likes})`
+          : '<ion-icon name="heart-outline"></ion-icon> (0)';
+    };
+    getLikesToItems()
+      .then(printLike)
+      .catch((e) => console.log(e));
+
+    btnRecipeElement.addEventListener('click', () => {
+      getItemData(food.idMeal);
+      hidenItems();
+    });
+    btnLikedElement.addEventListener('click', () => {
+      addLikeToItem(food.idMeal);
+      getLikesToItems()
+        .then(printLike)
+        .catch((e) => console.log(e));
+    });
+    board.appendChild(item);
```

Comment:
- [ ] When I click on the like button the like is added to the involvement API but the likes count is not updated So please fix that because the likes count suppose to be updated after adding a like.

Reply:
@Omar-Muhamad thankyou. I will ask for another review i appreciate your time.